Cannot install libx11 : compiler cannot create executables

Running `crew upgrade libx11` on x86_64, I have :
```shell
libx11: C interface to the X window system
https://x.org
version 1.6.12
Updating libx11...
No precompiled binary available for your platform, downloading source...
############################################################################################################################# 100.0%
Archive downloaded
Unpacking archive using 'tar', this may take a while...
gcc (GCC) 8.3.0
Copyright (C) 2018 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.

Building from source, this may take a while...
Rename all *.la files to *.la_tmp
checking build system type... x86_64-cros-linux-gnu
checking host system type... x86_64-cros-linux-gnu
checking for x86_64-cros-linux-gnu-gcc... x86_64-cros-linux-gnu-gcc
checking whether the C compiler works... no
configure: error: in `/usr/local/tmp/crew/libX11-1.6.12.tar.bz2.dir/libX11-1.6.12':
configure: error: C compiler cannot create executables
See `config.log' for more details
libx11 failed to install: `./configure --prefix=/usr/local --libdir=/usr/local/lib64 --mandir=/usr/local/share/man --build=x86_64-cros-linux-gnu --host=x86_64-cros-linux-gnu --target=x86_64-cros-linux-gnu` exited with 77
```
It successes if I remove lines 17/18 from libx11.rb
```ruby
 17     ENV['CFLAGS'] = "-fuse-ld=lld"                                                                                              
 18     ENV['CXXFLAGS'] = "-fuse-ld=lld" 
```
Same with `libxext`

--- a/packages/libx11.rb
+++ b/packages/libx11.rb
@@ -3,16 +3,16 @@ require 'package'
 class Libx11 < Package
   description 'C interface to the X window system'
   homepage 'https://x.org'
-  version '1.6.12'
+  version '1.6.12-1'
   compatibility 'all'
   source_url 'https://www.x.org/archive/individual/lib/libX11-1.6.12.tar.bz2'
   source_sha256 'f108227469419ac04d196df0f3b80ce1f7f65059bb54c0de811f4d8e03fd6ec7'
 
 
+  depends_on 'llvm' => ':build'
   depends_on 'xorg_proto'
   depends_on 'libxcb'
   depends_on 'libxtrans'
-  depends_on 'llvm' => ':build'
   
   def self.build
     ENV['CFLAGS'] = "-fuse-ld=lld"

